src/log.c

```diff
@@ -482,6 +482,7 @@ void log_counter_info(void)
 	log_info(" -> Unknown DNS queries: %i", counters->status[QUERY_UNKNOWN]);
 	log_info(" -> Unique domains: %i", counters->domains);
 	log_info(" -> Unique clients: %i", counters->clients);
+	log_info(" -> DNS cache records: %i", counters->dns_cache_size);
```

Comment:
What does "DNS cache records" tell us? How does it relate to the imported long-term data?

```
2024-01-04 22:11:36.590 [430399M] INFO: Imported 8396 queries from the long-term database
2024-01-04 22:11:36.598 [430399M] INFO:  -> Total DNS queries: 8396
2024-01-04 22:11:36.599 [430399M] INFO:  -> Cached DNS queries: 5696
2024-01-04 22:11:36.599 [430399M] INFO:  -> Forwarded DNS queries: 1889
2024-01-04 22:11:36.599 [430399M] INFO:  -> Blocked DNS queries: 35
2024-01-04 22:11:36.599 [430399M] INFO:  -> Unknown DNS queries: 4
2024-01-04 22:11:36.599 [430399M] INFO:  -> Unique domains: 583
2024-01-04 22:11:36.599 [430399M] INFO:  -> Unique clients: 3
2024-01-04 22:11:36.599 [430399M] INFO:  -> DNS cache records: 937
2024-01-04 22:11:36.599 [430399M] INFO:  -> Known forward destinations: 2

```

Reply:
Every unique client/domain combination creates a unique DNS cache record. Two queries for the same domain use one cache record. Two queries for the same domain by three clients create three cache records (because some clients may be blocking a domain while others don't due to groups). Thia is just yet another metric used by FTL for quite some time and which seemed useful to include here.